meta/system.py

```diff
@@ -388,7 +388,7 @@ def linear_alignment(self, name, corpus, flow, prefix="", **kwargs):
             self.crp[corpus], self.feature_flows[corpus][flow], **kwargs
         )
         j.add_alias(prefix + name)
-        self.mixtures[corpus][name] = j.mixtures
+        self.mixtures[corpus][name] = j.out_mixtures
         if hasattr(j, "alignment_path") and hasattr(j, "alignment_bundle"):
```

Comment:
should this also be `out_alignment_path` and `out_alignment_bundle`?

Ah you did not modify the `LinearAlignmentJob` intentionally?

Reply:
Oh, this `hasattr` stuff is dangerous...